docs/api.md

```diff
@@ -251,6 +278,29 @@ Some endpoints return further information when a session ID is given (e.g. `/api
 
 ## Etc.
 
+### Authorization
+
+Authorization is a simple form of privacy which prevents clients from interacting with the server API without being authorized to do so (usually manually, by a human). This limits interaction to specific users, which may be wanted so that the server is "private". It should be noted that **enabling authorization does not encrypt messages or user data**; it simply limits who can access that data via the API.
+
+Authorization is a server property and can only be enabled via the actual command line:
+
+```
+> set requireAuthorization on|off
+```
+
+When authorization is enabled for the first time, only admins will be verified. When a user is made to be an admin through the command line (`make-admin`), they will also be authorized. (If you're using an old database, you can authorize existing admins by running `make-admin` again.) Users can then be authorized via the [`authorize-user`](#post-apiauthorize-user) endpoint (in the official client, there's a dedicated settings page for this). Users can be deauthorized using [`deauthorize-user`](#post-apideauthorize-user).
```

Comment:
>If you're using an old database

Don't document this forever, thanks. Wait until we start doing releases before mentioning how to upgrade etc.